mirage/config.ts

```diff
@@ -68,10 +68,9 @@ export default function(this: Server) {
     this.namespace = '/api/v3/';
     this.get('/index-card-search', cardSearch);
     this.get('/index-value-search', valueSearch);
-    // this.get('/index-card/:id', Detail);
+    this.namespace = '/trove/';  // share's /api/v3/ is the same as /trove/
     this.get('/index-card-searches', cardSearch);
     this.get('/index-value-searches', valueSearch);
-    // this.get('/index-cards/:id', Detail);
```

Comment:
deduplicate -- can move the `/trove/` namespace up to replace the `/api/v3/` above (for the reason commented), and should remove the pluralized "searches"/"cards" routes (because the actual api 404s them)

Reply:
Ah, didn't realize the paths here are pluralized...